src/shared/bodies/complex_bodies/MeshHelper.h

```diff
@@ -0,0 +1,89 @@
+/* -------------------------------------------------------------------------*
+ *								SPHinXsys									*
+ * -------------------------------------------------------------------------*
+ * SPHinXsys (pronunciation: s'finksis) is an acronym from Smoothed Particle*
+ * Hydrodynamics for industrial compleX systems. It provides C++ APIs for	*
+ * physical accurate simulation and aims to model coupled industrial dynamic*
+ * systems including fluid, solid, multi-body dynamics and beyond with SPH	*
+ * (smoothed particle hydrodynamics), a meshless computational method using	*
```

Comment:
You can copy the credit claims from any header of the master, where there is no tab used.

Reply:
On my local machine it looks fine. However when I push it on the repository it gets distorted. I will change it.

![Screenshot 2024-04-09 075639](https://github.com/Xiangyu-Hu/SPHinXsys/assets/94981088/d40854b8-2042-42da-ba48-1f0a537c3be2)

Reply:
The reason is due to the tab which is usually defined differently by different systems. The new version in master does not used any tab.

Reply:
For file name, mesh_helper.h and mesh_helper.cpp would be better.

Reply:
In the latest commit I have updated the file name as suggested. Thank you for the feedback.